src/codegen/properties/functions.py

```diff
@@ -0,0 +1,75 @@
+"""Structure for storing function metadata from scrapigen."""
+from codegen.properties.adaptor_parameter import AdaptorParameter
+from codegen.properties.function_parameters import FunctionsParameter
+from codegen.properties.parameter import Parameter
+
+
+class Functions:
+    """Structure for storing function metadata from scrapigen."""
+
+    def __init__(self, function_name, function_metadata):
+        """Structure for storing function metadata from scrapigen."""
+        self._function_name = function_name
+        self._c_function_name = function_metadata["c_function_name"]
+        self._description = function_metadata["'description'"]
```

Comment:
The key name should not contain single quotes.

Reply:
Fixed the typos